modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -93,7 +93,7 @@ $ curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootim
 ** For installations on DASD-type disks, complete the following tasks:
 ... For `coreos.inst.install_dev=`, specify `/dev/dasda`.
 ... Use `rd.dasd=` to specify the DASD where {op-system} is to be installed.
-... Leave all other parameters unchanged.
+... You can adjust further parameters if required.
```

Comment:
Not sure if this should be a substep or separated (with a bullet)? Don't know enough about the feature to know if this should be included under the "task" heading above, but wanted to mention just in case.

Reply:
I understand your point. I'll keep as is if I move this one level up the bullet point would be after the examples, which would confuse the reader.